docs/getting-started/install-guide.md

```diff
@@ -4,7 +4,15 @@ This guide was last updated for v0.8.2.
 
 ---
 
-There are four methods for downloading and installing the Decred software. One is via dcrinstall (cross-platform), one is via the Windows Installer (Windows only of course, and the only way to get Paymetheus as of v0.8.2), another is via the precompiled binary releases (cross-platform), and the other is building the software yourself (cross-platform). The first three methods will be covered here and the fourth may be added at a later date.
+There are five methods for downloading and installing the Decred software. 
+
+* [The Windows Installer](#paymetheus) (.msi) - This only works for installing Paymetheus on Windows machines.
+* [dcrinstall](#dcrinstall) - A cross-platform, automatic installer/updater for the command-line applications (`dcrd`, `dcrwallet`, and `dcrctl`).
```

Comment:
Just a note to remind you, when v1.0.0 comes out, this will also install the promptsecret tool.  But since that isn't out yet, don't change this yet.